src/screens/IndexScreen.js

```diff
@@ -69,14 +81,25 @@ const getListItems = (query, data) => {
         data[query] &&
         data[query].map((newsItem) => ({
           id: newsItem.id,
-          subtitle: subtitle(
+          category: subtitle(
             momentFormat(newsItem.publishedAt),
             !!newsItem.dataProvider && newsItem.dataProvider.name
           ),
-          title:
+          name:
             !!newsItem.contentBlocks &&
             !!newsItem.contentBlocks.length &&
             newsItem.contentBlocks[0].title,
+          image: {
+            url:
+              !!newsItem.contentBlocks[0] &&
+              !!newsItem.contentBlocks[0].mediaContents &&
+              !!newsItem.contentBlocks[0].mediaContents.length &&
+              _filter(
+                newsItem.contentBlocks[0].mediaContents,
+                (mediaContent) =>
+                  mediaContent.contentType === 'image' || mediaContent.contentType === 'thumbnail'
+              )[0].sourceUrl.url,
+          },
```

Comment:
the change of the keys needs tobe reverted for news items, to have the default back again.